src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -488,22 +490,12 @@ namespace OpenLoco::Ui::WindowManager
 
     size_t indexOf(const Window& pWindow)
     {
-        int i = 0;
-
-        for (Ui::Window* w = &_windows[0]; w != _windowsEnd; w++)
-        {
-            if (w == &pWindow)
-                return i;
-
-            i++;
-        }
-
-        return 0;
+        return &pWindow - _windows.data();
```

Comment:
Just for my understanding, would this not need to be divided by `sizeof(Window)`?

Reply:
No, it gives the distance by pointer type and not by bytes.

Reply:
Alright, nice.